web/src/ResourceList.css

```diff
@@ -1,35 +1,41 @@
-.ResourceList {
-  margin: 1em 1.5em;
-  font-size: 1.4em;
+.resources * {
```

Comment:
should we create a `cell` class? this is semantically kind of hard to understand